### ISSUE TITLE
v-menu in app-bar needs two clicks to show after navigating to another page

### ISSUE THREAD

**Issue by micheldenegri** [nuxt-community/vuetify-module#282]
**Module version**
1.10.3

**Describe the bug**
When navigation to different routes and you try to open a v-menu in the v-app-bar a weird refresh occurs and the menu does not open until you click for the second time. 

**To Reproduce**
https://codesandbox.io/s/toolbar-dropdown-nuxtjs-vuetify-mqbcu

1. Refresh page
2. Click on Second Page in AppBar
3. Click on Dropdown (nothing happens)
4. Click on Dropdown again (Dropdown is shown)

**Expected behavior**
After navigating to another page dropdown will show menu on first click with no refresh.

**Comment by praburangki** [comment]
I believe this only happens in development?
I've encountered this issue for a while but only on dev env.

**Comment by micheldenegri** [comment]
You are right, I'm not able to recreate this with a production build. Must be something related dev hot module reload perhaps?

**Comment by kevinmarrec** [comment]
Really weird issue indeed.. I think kinda 75% issues of this module are related to something that is unfixable here, and it make some of these behaviors happen, most likely it's how components are loaded (as well as their respective css) and the load order.

The thing is that Nuxt module doesn't do more that registering the Vuetify library, so it may still be some issue around Vuetify code itself (probably when using `vue-meta`)
